### ISSUE TITLE
TypeError on link Click Event

### ISSUE THREAD

**Issue by scottkatzelnick** [LonelyCpp/react-native-youtube-iframe#139]
**Describe the bug**
Any touch (click) event on a link on the YouTube player results in error:

<pre>
  <code>
    TypeError: undefined is not an object (evaluating 'request.mainDocumentUrl.startsWith')
  </code>
</pre>

*
**For Android, I get the visible error; however, for iOS devices, the error is only shown in the console**

**To Reproduce**
I'm using the standard implementation as described on the website.

  ```javascript
return (
  <View style={{ height: styleDims.height, overflow: 'hidden' }}>
      <YoutubePlayer
          width={styleDims.width}
          height={styleDims.height}
          videoId={youtubeId}
      />
  </View>
)
```

**Expected behavior**
Opens either default browser or native App to video on YouTube

**Screenshots**

### Android
![Screenshot_1619813186](https://user-images.githubusercontent.com/32975303/116748871-1542f980-a9ce-11eb-9a57-735bca8ab2ec.png)

### iOS
<img width="430" alt="Screen Shot 2021-04-30 at 4 14 05 PM" src="https://user-images.githubusercontent.com/32975303/116749546-1aed0f00-a9cf-11eb-9ab2-564f330213c3.png">

**Smartphones (please complete the following information):**
 - Device: Pixel 3 XL
 - OS + version:  Android 11
 - `react-native-youtube-iframe` version "^2.0.1" 
 - `react-native-webview` version "8.1.1"
 - `Expo` verison "^37.0.0" (Detached)

 - Device: iPhone 12 Pro Max
 - OS + version:  iOS 14.5 Build 18E182)
 - `react-native-youtube-iframe` version "^2.0.1" 
 - `react-native-webview` version "8.1.1"
 - `Expo` verison "^37.0.0" (Detached)

I am able to use the controls normally and the error doe not stop playback if a video happens to be playing when I click the links. Just want to make sure this error is not affecting the player in any unforeseen way before I push to production. I could catch it with the onError prop and just () => {return} if need be.

**Comment by LonelyCpp** [comment]
yep, I've fixed it in here - https://github.com/LonelyCpp/react-native-youtube-iframe/commit/7d548c8dba911837dc7e974dfa15fa0d9de828d0

I'll make a patch release to npm soon

**Comment by scottkatzelnick** [comment]
> yep, I've fixed it in here - https://github.com/LonelyCpp/react-native-youtube-iframe/commit/7d548c8dba911837dc7e974dfa15fa0d9de828d0
> 
> 
> 
> I'll make a patch release to npm soon

Great! Looking forward to the update.

**Comment by LonelyCpp** [comment]
fixed in `v2.1.0` 🥳 

reopen this if you still face the same problem

**Comment by scottkatzelnick** [comment]
Thanks! That’s great news!
